uktrade/data-hub-frontend
# Tidy up Investments collection list pages

## Description of change

Since the collection lists were converted into React we need to remove and re-organise a few files. We simply need to delete/modify some of the old files, these files will typically be middleware and transformers that were originally used in the node layer. Any data transformations we still use need to be moved into the client area and exported using es6 modules syntax. 

I also noticed some transformations happening in reducers so I have moved this closer to the API so they now sit alongside the task files.

## Test instructions
All investments collection list should function as before

1. View Investments projects `/investments/projects`
2. View Investments profiles /investments/profiles`
3. View UK opportunities `investments/opportunities`
4. View opportunities details page `/investments/opportunities` then click on an opportunity

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
